Hi

Gemk83 opened the pull request:
Love

ysy717324540 committed:
update chain

LiYuqing-2020 commented:
{
    "name": "UN Chain",
    "chainId": 1681681688,
    "shortName": "un",
    "chain": "UN",
    "network": "mainnet",
    "networkId": 1681681688,
    "nativeCurrency": {
        "name": "UN",
        "symbol": "UN",
        "decimals": 18
    },
    "rpc": [
        "http://116.62.244.90:50121",
        "ws://116.62.244.90:60121"
    ],
    "faucets": [],
    "infoURL": "116.62.244.90:40012",
    "app_resource": {
        "ic_chain_select": "https://files.moacchina.info/2021/12/14/e618eec7-4fb0-4210-bbee-6d0cee2c1650_1.png",
        "ic_chain_unselect": "https://files.moacchina.info/2021/12/14/ede37514-9792-443b-897f-01d1377256e0_2.png",
        "color_chain_bg": "0x58B2AF"
    }}

LiYuqing-2020 committed:
Update chains.json

chendatony31 committed:
Merge pull request #33 from LiYuqing-2020/patch-2

Update chains.json

Emrediyorsun commented:
EmreKspToken

Emrediyorsun commented:
EmreKspcoin

giki0318 committed:
Update chains.json

add aurora network

giki0318 committed:
Update chains.json

delete old near data

Commit:
add vision network

Commit:
Update chains.json

chendatony31 committed:
Merge pull request #36 from vchain007/pr/vision-network

add vision mainnet and testnet

Commit:
Update chains.json

decimals 6 --> 18

native vs decimal is 6 ,but ethereum compatible interface already adapt

yuop2002cn committed:
Add SAT L2 network

chendatony31 committed:
rename Goerli

heipacker committed:
Merge pull request #37 from vchain007/patch-1

vision - Update chains.json

heipacker committed:
Merge pull request #38 from yuop2002cn/patch-1

Add SAT L2 network

chendatony31 committed:
format

chendatony31 committed:
add evmos

chendatony31 committed:
Update README.md

chendatony31 committed:
update build website

chendatony31 committed:
add transit logo

chendatony31 committed:
merge https://chainid.network/chains.json

chendatony31 committed:
add tokenpocket & dfox

chendatony31 committed:
update kcc

viaweb3 committed:
fix KCC networkId

viaweb3 committed:
add KCC explorers

chendatony31 committed:
Merge pull request #43 from lingyun92/main

fix KCC networkId

Pana committed:
Add info about conflux chain

chendatony31 committed:
Merge pull request #45 from conflux-fans/main

Add info about conflux chain

chendatony31 committed:
format

chendatony31 committed:
add trust testnet

RunsStorm committed:
add MTC info

chendatony31 committed:
Merge pull request #49 from RunsStorm/main

add MTC info

chendatony31 committed:
update cro

chendatony31 committed:
dfox->tokenpocket

chendatony31 committed:
update platon

chendatony31 committed:
sync ethereum-lists

chendatony31 committed:
add darkmode

chendatony31 committed:
add cube

chendatony31 committed:
add cube

chendatony31 committed:
sync networklist

chendatony31 committed:
add canto

chendatony31 committed:
add ethw & etf

chendatony31 committed:
ethf rename

chendatony31 committed:
add FON smart chain

giki0318 committed:
Update chains.json

add fon resource

giki0318 committed:
Update README.md

wangxihaowxh committed:
Update chains.json

update chains.json

giki0318 committed:
Update chains.json

change name

opera0001 committed:
Merge pull request #75 from wangxihaowxh/main

Update chains.json

chendatony31 committed:
format bug fix

chendatony31 committed:
remove KorthoTest

giki0318 committed:
update info

chendatony31 committed:
update order

chendatony31 committed:
add core

wx23xw commented:
r

chendatony31 committed:
sync chainid.network

chendatony31 committed:
move zksync network

dglw committed:
first

chendatony31 committed:
Merge pull request #79 from dglw/main

add AlveyChain Mainnet

giki0318 committed:
change RPC for pulsechain

giki0318 committed:
add chain info

f4nzkrys31 committed:
Update chains.json

添加自定义网络（便捷加链） PEGO Mainnet

opera0001 committed:
Merge pull request #81 from f4nzkrys31/patch-1

Update chains.json

f4nzkrys31 committed:
Update chains.json

update selected/unselected logo url, previous url has expired.

chendatony31 committed:
Merge pull request #82 from f4nzkrys31/f4nzkrys31-update-logo

Update chains.json

chendatony31 committed:
update mapo

sammvio committed:
Update README.md

在简介新增域名：https://chainlist.tokenpocket.pro/

sammvio committed:
Update README.md

在简介添加域名：https://chainlist.tokenpocket.pro/

giki0318 committed:
Update README.md

change some intro

chendatony31 committed:
sync chainlist

fadeev committed:
Update ZetaChain Athens Testnet details

chendatony31 committed:
Merge pull request #86 from fadeev/main

Update ZetaChain Athens Testnet details

jasonzysun committed:
Modified rpc information, browser information, and faucet information in the Qitmeer Network mainnet(813) and testnet(8131) configuration

jasonzysun committed:
Modified browser url in the Qitmeer Network mainnet(813)

ETENetwork committed:
Update chains.json

ETENetwork committed:
Update chains.json

chendatony31 committed:
Merge pull request #87 from jasonzysun/main

Modified qitmeer network config

opera0001 committed:
Merge pull request #88 from ETENetwork/patch-1

Add ETE Chain Mainnet

chendatony31 committed:
fix format

chendatony31 committed:
update fon

sammvio committed:
Update chains.json (Zetachain Testnet logo)

Update zetachain testnet logo

chendatony31 committed:
Merge pull request #89 from TP-Lab/sammvio-patch-1

Update chains.json (Zetachain Testnet logo)

chendatony31 committed:
fix format

sammvio committed:
Update chains.json

chendatony31 committed:
Merge pull request #90 from TP-Lab/sammvio-patch-2

Update chains.json

chendatony31 committed:
update fon image

giki0318 committed:
update tomochain to viction

jihuiling committed:
Update chains.json

https://bscscan.com/tx/0xb2731df366db70bab4cc03b729c925924d2a284878c429c09f0604bef2968516

jihuiling committed:
Update chains.json

jihuiling committed:
Update chains.json

图片尺寸和颜色已经更改好了

jihuiling committed:
Update chains.json

jihuiling committed:
Update chains.json

修改未选中照片

jihuiling committed:
Update chains.json

already update

opera0001 committed:
Merge pull request #92 from jihuiling/patch-1

Update chains.json

chendatony31 committed:
format

jihuiling committed:
Update PKB logo

chendatony31 committed:
sync chainlist

Gemk83 reviewed (commented):
Done

opera0001 committed:
Merge pull request #94 from jihuiling/patch-2

Update PKB logo

Gemk83 reviewed (commented):
Done

jihuiling committed:
Update logo

opera0001 committed:
Merge pull request #95 from jihuiling/patch-4

Update logo

yiweichi committed:
add zkfair network

chendatony31 committed:
Merge pull request #96 from ZKFair/main

Add zkfair network

Commit:
Change to AIA chain

dglw committed:
AlveyChain Mainnet RPCs Updated

giki0318 committed:
Merge pull request #97 from moumc/main

Change to AIA chain

opera0001 committed:
Merge pull request #98 from dglw/main

AlveyChain Mainnet RPCs Updated

servis committed:
add aia chain app_resource

chendatony31 committed:
add blast

chendatony31 committed:
sync chainlist

giki0318 committed:
Add logos to Blast Sepolia

chendatony31 committed:
update taiko

Commit:
update the chain infos

opera0001 committed:
Merge pull request #101 from mapprotocol/main

update the chain infos for the map protocol

sammvio committed:
Update chains.json

added merlin mainnet info

chendatony31 committed:
format

chendatony31 committed:
sync chainlist

chendatony31 committed:
sort

chendatony31 committed:
update blast

clark-superman committed:
Update chains.json

add WDC Chain

opera0001 committed:
Merge pull request #103 from clark-superman/main

Update chains.json

giki0318 committed:
add berachain & endurance app resource

giki0318 committed:
add more sample code

clark-superman committed:
Update chains.json

change logo links. sry

giki0318 committed:
update morph testnet info

giki0318 committed:
add zora&mode network resource

giki0318 committed:
Merge pull request #104 from clark-superman/main

Update chains.json

giki0318 committed:
update resources

giki0318 committed:
add BounceBit Testnet

giki0318 committed:
add zklink nova

giki0318 committed:
change order of the list

chenjia404 committed:
Update chains.json

add rpc

giki0318 committed:
add Frax chain

giki0318 committed:
change zklink nova brand assets

giki0318 committed:
add cycle testnet

opera0001 committed:
Merge pull request #105 from qkidev/main

Update chains.json add rpc

giki0318 committed:
add mint sepolia testnet

chendatony31 committed:
format

chendatony31 committed:
Merge pull request #99 from aiachain/main

add aia chain app_resource

giki0318 committed:
add lightlink resources

chendatony31 committed:
remove no rpc chain

chendatony31 committed:
sync chainlist

giki0318 committed:
add shortname for chainid 20028

Tixoncoinl-aboratory committed:
tixoncoin

Tixoncoinl-aboratory committed:
tixoncoin_fixes

opera0001 committed:
Merge pull request #108 from qukualian/tixoncoin_fixes

Tixoncoin fixes

giki0318 committed:
update resources

chendatony31 committed:
add Matchain

giki0318 committed:
add app_resource for matchain.

PcdDao committed:
Update chains.json

Add Phecda Chain info
donation hex https://bscscan.com/tx/0xb33e06fd09bf5f8801d99ec9a2d2e79a28fe78fe34bc7639493592d701a4bfc1

opera0001 committed:
Merge pull request #112 from PcdDao/main

Update Phecda chains.json

giki0318 committed:
change file resources

LunaraAdemira committed:
Add MAK Chain

LunaraAdemira committed:
Add MAK Chain

LunaraAdemira committed:
Add MAK Chain

opera0001 committed:
Merge pull request #121 from LunaraAdemira/main

Add MAK Chain

giki0318 committed:
restore network files.

Orlanduis commented:
const ethereum anticripto= { 
        id: 19845,
        name: "anticripto network",
        network: "Ethereum",
        nativeCurrency: eth
          decimals: 18,
          name: "anticripto network",
          symbol: "eth",
        },
        rpcUrls: {
          public: { http: ["https://virtual.mainnet.rpc.tenderly.co/a6f2e2a3-1e3a-4f14-93df-69fda2461eb4"] },
          default: { http: ["https://virtual.mainnet.rpc.tenderly.co/c97b21ff-b8c2-4a2b-b99f-52e09f6537b1"] },
        },
        blockExplorers: {20724265
          etherscan: {
            name: "anticripto network",
            url: "https://etherscan.io",
          },
          default: {
            name: "anticripto network",
            url: "https://etherscan.io",
          },
        },
      } as const satisfies Chain;
      
  const { chains, publicClient, webSocketPublicClient } = configureChains(
        [ethereum anticripto],
        [
          jsonRpcProvider({
            rpc: (chain) => ({https://virtual.mainnet.rpc.tenderly.co/c97b21ff-b8c2-4a2b-b99f-52e09f6537b1
              http: "https://virtual.mainnet.rpc.tenderly.co/a6f2e2a3-1e3a-4f14-93df-69fda2461eb4",
            }),
          }),
        ]
      );

giki0318 committed:
update resource

whyubel1eve committed:
add Unichain Sepolia

chendatony31 committed:
Merge pull request #122 from whyubel1eve/patch-1

Add Unichain Sepolia

chendatony31 committed:
update format

whyubel1eve committed:
Add World Chain

chendatony31 committed:
Merge pull request #125 from whyubel1eve/patch-1

Add World Chain

chendatony31 committed:
add native logo

sammvio committed:
Update chains.json - ApeChain

chendatony31 committed:
Merge pull request #126 from TP-Lab/sammvio-patch-4

Update chains.json - ApeChain

whyubel1eve committed:
Update ApeChain

whyubel1eve committed:
Update apecoin logo

chendatony31 committed:
Merge pull request #127 from whyubel1eve/patch-2

Update ApeChain

whyubel1eve committed:
Add  Morphl2

chendatony31 committed:
Merge pull request #128 from whyubel1eve/patch-3

Add  Morphl2

whyubel1eve committed:
Add exsat

chendatony31 committed:
Merge pull request #129 from whyubel1eve/patch-3

Add exsat

chendatony31 committed:
format

giki0318 committed:
correct json

giki0318 committed:
remove folder

chendatony31 committed:
add eslint

chendatony31 committed:
update file name

chendatony31 committed:
add Gravity

chendatony31 committed:
Merge pull request #131 from chendatony31/main

add Gravity

whyubel1eve committed:
Update Gravity

chendatony31 committed:
Merge pull request #133 from whyubel1eve/patch-6

Update Gravity

whyubel1eve committed:
Add Lumia Prism

whyubel1eve committed:
fix: typo

chendatony31 committed:
Merge pull request #134 from whyubel1eve/patch-7

Add Lumia Prism

whyubel1eve committed:
Update RPCs of Flare Mainnet

chendatony31 committed:
Merge pull request #135 from whyubel1eve/main

Update RPCs of Flare Mainnet

whyubel1eve committed:
Add SlerfChain

chendatony31 committed:
Merge pull request #136 from whyubel1eve/patch-3

Add SlerfChain

chendatony31 committed:
add duckchain

Idcofficekaiyuan committed:
Update IDC chains.json

opera0001 committed:
Merge pull request #142 from Idcofficekaiyuan/main

Update IDC chains.json

giki0318 committed:
change chain assets

chendatony31 committed:
format

chendatony31 committed:
update order

giki0318 committed:
add ink chain

giki0318 committed:
Merge branch 'main' of github.com:TP-Lab/networklist-org into main

# Conflicts:
#	chains.json

giki0318 committed:
add resources for ink

giki0318 committed:
add hype evm testnet

giki0318 committed:
modify duckchain resource

giki0318 committed:
change DuckChain text and textbg color

giki0318 committed:
sort list

gs2040457 committed:
Add Akashic Chain

https://bscscan.com/tx/0xb33e06fd09bf5f8801d99ec9a2d2e79a28fe78fe34bc7639493592d701a4bfc1

opera0001 committed:
Merge pull request #148 from gs2040457/main

Add Akashic Chain

giki0318 committed:
Add Soneium

Setland34 reviewed (approved):
PR Approval

Setland34 reviewed (approved):
Looks Good

giki0318 committed:
Update chains.json

add unichain

giki0318 committed:
Merge pull request #150 from giki0318/main

Update chains.json

giki0318 committed:
change unichain native currency logo

DalethLinsley committed:
Update chains.json

DalethLinsley committed:
Update chains.json

giki0318 committed:
Merge pull request #153 from DalethLinsley/main

HyperEVM

giki0318 committed:
add hyperEVM

giki0318 committed:
Merge branch 'main' of github.com:TP-Lab/networklist-org into main

giki0318 committed:
solve conflicts

giki0318 committed:
change text and bg color

giki0318 committed:
color changed

giki0318 committed:
add monad testnet

chendatony31 committed:
sync chainlist

DalethLinsley committed:
Update chains.json

test

DalethLinsley committed:
Update chains.json

Add Vana Mainnet

DalethLinsley committed:
Update chains.json

giki0318 committed:
Update chains.json

fix issues

giki0318 committed:
Merge pull request #157 from DalethLinsley/main

Vana Mainnet

chendatony31 committed:
sync chainlist

chendatony31 committed:
add cryptoZR

DalethLinsley committed:
Update chains.json

giki0318 committed:
Update chains.json

fix errors

giki0318 committed:
Merge pull request #162 from DalethLinsley/main

Add ENI Mainnet

giki0318 committed:
fix merge error

giki0318 committed:
fix conflict and re-add the info

giki0318 committed:
change resources

mixmax2022dev committed:
Update chains.json

opera0001 committed:
Merge pull request #169 from mixmax2022dev/main

Update chains.json

chendatony31 committed:
add Mova Chain

DankFang committed:
add Nexus Chain

opera0001 committed:
Merge pull request #176 from DankFang/main

add Nexus Chain

fdkgkdf2 committed:
Update chains.json

fdkgkdf2 committed:
Update chains.json

opera0001 committed:
Merge pull request #178 from fdkgkdf2/main

feat: add MCNChain to network list Update chains.json

czchangsheng-dot committed:
Add AOU Chain to chains.json

Add AOU Chain testnet configuration.

Chain name: AOU Chain
Chain ID: 2236
Native token: AOU
Decimals: 18
RPC: https://testnet-rpc.aouscan.com
Explorer: https://testnet.aouscan.com
Official website: https://www.aoulink.io
Logo: https://www.aoulink.io/aou.png

Payment transaction hash:
https://bscscan.com/tx/0x0014980bcc17cec2be54f7403d39565ef6587138e6a840b49ca49bcaa749ae1d

Message transaction hash:
https://bscscan.com/tx/0xb6be7030b0e6a53a0860561af21b759d5b0071bfa44866985982e15a9a1df4de

opera0001 committed:
Merge pull request #186 from czchangsheng-dot/main

Add AOU Chain to chains.json
